lib/modules/datasource/maven/index.spec.ts

```diff
@@ -1,16 +1,25 @@
+import * as _googleAuth from 'google-auth-library';
```

Comment:
```suggestion
import { GoogleAuth as _googleAuth } from 'google-auth-library';
```

You want to mock only the constructor, there is no need to import everything.